src/main/kotlin/org/gradlewebtools/minify/minifier/result/Event.kt

```diff
@@ -21,10 +21,10 @@ open class Event {
         charNo = error.charno
```

Comment:
Could this and the lineNumber create the same issue?

Reply:
I guess everything from the "Java World", Yes in theory.

Reply:
I try to look into Closure for that...

Reply:
I think if its only in theory we dont have to address it.
THe example code would help.

Reply:
```java
public abstract class JSError implements Serializable {

  /** A type of the error. */
  public abstract DiagnosticType getType();

  /** Description of the error. */
  public abstract String getDescription();

  /** Name of the source */
  @Nullable
  public abstract String getSourceName();

  /** One-indexed line number of the error location. */
  public abstract int getLineno();

  /** Zero-indexed character number of the error location. */
  public abstract int getCharno();
```
As you can see only "getSourceName" is marked  with @Nullable, I think we should therefor trust them not to return null for the other members like "getDescription" and "getLineno".

Reply:
Very nice documentation of them

Reply:
> I think if its only in theory we dont have to address it.
> THe example code would help.

Yes, I think we can trust them following nullability annotation convention